src/settings.js

```diff
@@ -1407,6 +1407,9 @@ var CYBERDWARF = 0;
 // Path to the CyberDWARF debug file passed to the compiler
 var BUNDLED_CD_DEBUG_FILE = "";
 
+// Temporary setting to force generation and preservation of full DWARF debug info
+var FULL_DWARF = 0;
```

Comment:
this can go in `src/settings_internal.js` (unless we intend for users to pass it directly, which I don't think we do?)

Reply:
No, we don't. I tried to use just a direct setting instead of a -g flag, but that's apparently not checkable at the time you need it (i.e. when you are parsing the -g flags, or right after parsing all the flags). Maybe I just wasn't putting it in the right place :man_shrugging: